### ISSUE TITLE
Usage instructions and example give "can't find crate for `msgpack`"

### ISSUE THREAD

**Issue by pwaller** [3Hren/msgpack-rust#89]
I'm new to rust. I followed the instructions in the README to build the example given there. I'm running a rust nightly.

The build gives this warning, which seems like it might by relevant: (full output below)

```
dependency (rmp) specified without providing a local path, Git repository, or version to use. This will be considered an error in future versions
```

# My `rustc --version`:

```
$ rustc --version
rustc 1.14.0-nightly (5665bdf3e 2016-11-02)
```

# My `Cargo.toml`:

```toml
[package]
name = "msgpack-hello-world"
version = "0.0.0"

[dependencies.rmp]
rmp = "^0.7"
```

# Running `cargo build`

```
$ cargo build
warning: dependency (rmp) specified without providing a local path, Git repository, or version to use. This will be considered an error in future versions
warning: unused manifest key: dependencies.rmp.rmp
   Compiling msgpack-hello-world v0.0.0 (file:///home/pwaller/Projects/msgpack-hello-world)
error[E0463]: can't find crate for `msgpack`
 --> src/main.rs:1:1
  |
1 | extern crate rmp_serialize as msgpack;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: aborting due to previous error

error: Could not compile `msgpack-hello-world`.

To learn more, run the command again with --verbose.
```

**Comment by euclio** [comment]
Hi @pwaller, check out the changes to the README in my PR. Hopefully they clear things up?
